### DISCUSSION TITLE
Copilot access after re-verifying teacher status

### DISCUSSION THREAD

**Discussion by lhebates** [community/community#73331] [GitHub Education]
### Select Topic Area

Question

### Body

I was able to use copilot during the past year. But two weeks ago, I lost access and was prompted to re-verify my teacher credentials, which I did and was approved. But the access to copilot was never restored. Are educator access to copilot still available? If so, how should I proceed? Thank you!

**Comment by GigaSmurf** [comment]
Github Copilot should still be available for educators and students through GitHub's Education program. If you've re-verified your teacher credentials and your access to GitHub Copilot hasn't been restored, it could be due to a number of reasons, including processing delays or system issues.

Maybe double-check your Github education status to ensure your teacher benefits are fully active or check your profile in the top right and see what it says on the Copilot tab. 

If you've tried everything, then you can contact Github support. Use their dedicated contact forms for education-related queries.

**Reply by Marcysiegel** [reply]
Hi

**Reply by lhebates** [reply]
Thanks. My education status shows "approved", but the copilot setting page only shows "start free trial". I tracked down the contact form and filled it out. Hopefully will hear back soon.

**Reply by Marcysiegel** [reply]
Hello my name is marcy and you?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @lhebates! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by Marcysiegel** [reply]
OK you can add me up @Gimenez71

**Comment by lhebates** [comment]
I am happy to report that my copilot access is not re-enabled. It has been about two weeks since the re-verification though. I also filed a ticket, but haven't received any response there. The access might just take that long, just in case others ran into similar situations.
